lib/phctl.rb

```diff
@@ -237,12 +237,18 @@ def costreport_workflow
       run_workflow(components)
     end
 
-    desc "overlap ORGANIZATION", "Generate an overlap report for the given organization"
+    desc "overlap ORGANIZATION [--matching-members-count", "Generate an overlap report for the given organization"
+    option :matching_members_count, type: :boolean
```

Comment:
Might consider adding a desc to this one:
```
option :matching_members_count, type: :boolean, desc: "Include count of HT members that report holdings for each item"
```

(one could probably greatly improve that desc I came up with)

and maybe even a `default: false` although I don't really like the way Thor formats boolean defaults. Since you have the optionality in the command desc I don't think it's needed